documentation/guide-beanmapping.asciidoc

```diff
@@ -21,16 +21,29 @@ public UserEto mapUser(UserEntity source) {
 Therefore we are using a `BeanMapper` for this purpose that makes our lives a lot easier.
 
 == Bean-Mapper Dependency
-To get access to the `BeanMapper` we use this dependency in our POM:
+To get access to the `BeanMapper` we have to use either of below dependency in our POM:
+
+We recommend orika for new projects but still support dozer ,please see https://www.baeldung.com/java-performance-mapping-frameworks#2-orika
```

Comment:
`We started with dozer.sourceforge.net/[dozer] in devon4j and still support it. However, we now recommend https://orika-mapper.github.io/orika-docs/[orika] (for new projects) as it is much faster (see https://www.baeldung.com/java-performance-mapping-frameworks#2-orika[Performance of Java Mapping Frameworks]).`